src/main/java/pages/components/BaseComponent.java

```diff
@@ -0,0 +1,31 @@
+package pages.components;
+
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AppiumFieldDecorator;
+import io.github.cdimascio.dotenv.Dotenv;
+import org.openqa.selenium.support.PageFactory;
+import utils.BrowserManagement;
+import utils.Wait;
+
+import java.time.Duration;
+
+
+public class BaseComponent {
+
+    protected AppiumDriver<MobileElement> appiumDriver;
+    protected Wait wait;
+    protected Dotenv dotenv;
+    protected int timeOutSeconds;
```

Comment:
If this is a constant, maybe you can declare it as a private static int and avoid using dotenv for this ;)

Reply:
I did not place it static because sometimes the resources, such as ram and processing, used to create the test machines, can vary and affect the performance of the tests, in which case, the waiting times should be easily changed according to the machine resources.

Reply:
That makes sense!